taskcluster/ci/ui-test/kind.yml

```
@@ -64,3 +64,16 @@ jobs:
                 - [automation/taskcluster/androidTest/ui-test.sh, x86-screenshots-tests, app.apk, android-test.apk, '-1']
         treeherder:
             symbol: debug(screenshots-x86)
+    x86-beta:
+        attributes:
+            build-type: beta
```

Nit:

```suggestion
            build-type: beta-firebase
```

Let's match the build type we defined here:
https://github.com/mozilla-mobile/fenix/blob/7a08175568fe2a7e426b8542554a3a295aab5189/taskcluster/ci/build/kind.yml#L57